src/_scss/pages/generateDetachedFiles/_fyPicker.scss

```diff
@@ -62,6 +63,7 @@
         right: 0;
```

Comment:
`fy-picker__list` needs a z-index to prevent overlap 
<img width="297" alt="screen shot 2018-12-12 at 3 58 39 pm" src="https://user-images.githubusercontent.com/7108785/49898458-18ea7b80-fe27-11e8-9d88-7e1aada17e08.png">

Reply:
Fixed, Thanks!